src/hearings/pipes/hearing-answers.pipe.spec.ts

```diff
@@ -127,14 +127,14 @@ describe('HearingAnswersPipe', () => {
 
   it('should transform case flag', () => {
     const result$ = hearingAnswersPipe.transform(AnswerSource.CASE_FLAGS, of(STATE), 0);
-    const caseFlags = '<strong class=\'bold\'>Jane Smith</strong>\n<ul><li>Sign Language Interpreter</li><li>Hearing Loop</li><li>Larger font size</li><li>Reading documents for customer</li><li>Sign Language Interpreter</li><li>Language Interpreter</li></ul><br>';
+    const caseFlags = '<strong class=\'bold\'>Jane Smith</strong>\n<ul><li>Sign Language Interpreter</li><li>Hearing Loop</li><li>Larger font size</li><li>Reading documents for customer</li><li>Sign Language Interpreter</li></ul><br>';
```

Comment:
I don't think this can be right - We shouldn't have Sign Language Interpreter twice

Reply:
@andywilkinshmcts 
It's been there from long and the reason behind duplication is because of duplication in the 
reasonableAdjustments: [
            'RA0042',
            'RA0053',
            'RA0013',
            'RA0016',
            'RA0042',
            'RA0009'
          ]
not sure why but if I remove it then other tests are failing. so not doing the change. 
As far as this change is concerned Language Interpreter is not getting added if reasonable adjustment has Sign Language Interpreter.